### DISCUSSION TITLE
[general] Some suggestions on `tar_rep()`

### DISCUSSION THREAD

**Discussion by psychelzh** [ropensci/tarchetypes#195] [General]
### Help

- [X] I understand and agree to <https://books.ropensci.org/targets/help.html>.

### Description

Thanks for adding `tar_rep()` and `tar_rep2()`, which are very convenient. But I found several things not so well-fitted in my practice:

1. Is it possible to store `tar_rep`, `tar_batch`, and `tar_seed` in meta only? Or store them as attributes of the generated objects?
2. If storing them is a must-have, I am wondering if we should use "." prefix to minimize conflicts with user's data?
3. The documentation does not mention the store `tar_seed`, which should be a small typo to fix.

**Comment by psychelzh** [comment]
The first needs some considerations when the generated objects are `NULL`.

**Reply by wlandau** [reply]
> Is it possible to store tar_rep, tar_batch, and tar_seed in meta only? Or store them as attributes of the generated objects?

I prefer these to be explicit so users can more easily find them.

> If storing them is a must-have, I am wondering if we should use "." prefix to minimize conflicts with user's data?

Maybe the names should have started with "." from the beginning, but at this point that would be a breaking change.

> The documentation does not mention the store tar_seed, which should be a small typo to fix.

Added in 0d947475115b312e86deeedad8213321dde72379.
